v0.3/cruorg/components/accordion/accordion.php

```diff
@@ -0,0 +1,74 @@
+<?php function cru_accordion($color)
+{ ?>
+  <div class="accordion cru-accordion-<?= $color; ?>">
+    <div class="cmp-accordion">
+      <?php for ($i = 1; $i <= 3; $i++) : $state = $i == 1 ? 'expanded' : 'hidden'; ?>
+        <div class="cmp-accordion__item">
+          <h3 class="cmp-accordion__header">
+            <button class="cmp-accordion__button<?php if ($i == 1) {
+                                                  echo ' cmp-accordion__button--expanded';
+                                                } ?>">
+              <span class="cmp-accordion__title">Item <?= $i ?></span>
+              <span class="cmp-accordion__icon">
+                <svg width="10" height="6" viewBox="0 0 10 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M9 0.988281L5.236 4.66802C5.17349 4.72911 5.08872 4.76343 5.00033 4.76343C4.91195 4.76343 4.82718 4.72911 4.76467 4.66802L1 0.988281" stroke="#383F43" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+              </span>
+            </button>
+          </h3>
+          <div class="cmp-accordion__panel cmp-accordion__panel--<?= $state ?>">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Eu mi bibendum neque egestas congue quisque egestas. Varius morbi enim nunc faucibus a pellentesque. Scelerisque eleifend donec pretium vulputate sapien nec sagittis.
+          </div>
+        </div>
+      <?php endfor; ?>
+    </div>
+  </div>
+<?php } ?>
```

Comment:
@sebamelzi Please restore my function back to the way it was. There's no need for "white" and "black" classes. The "white" class styles should simply be the default.

And the component markup from AEM doesn't use SVGs, so there's no need for all that.

There's no need for a second function either. If you need another optional class, add it as an optional argument to the function. If you need help with that, let me know.